crates/fuel-core/src/executor.rs

```diff
@@ -2764,4 +2800,242 @@ mod tests {
         let receipts = &tx_status[0].receipts;
         assert_eq!(time.0, receipts[0].val().unwrap());
     }
+
+    #[cfg(feature = "relayer")]
+    mod relayer {
+        use super::*;
+        use crate::database::RelayerReadDatabase;
+        use fuel_core_relayer::storage::History;
+        use fuel_core_storage::{
+            tables::{
+                FuelBlocks,
+                SpentMessages,
+            },
+            transactional::Transaction,
+            StorageAsMut,
+        };
+
+        fn database_with_genesis_block(da_block_height: u64) -> Database {
+            let db = Database::default();
+            let mut block = Block::default();
+            block.header_mut().set_da_height(da_block_height.into());
+            block.header_mut().recalculate_metadata();
+
+            let mut db_transaction = db.transaction();
+            db_transaction
+                .as_mut()
+                .storage::<FuelBlocks>()
+                .insert(&0.into(), &block)
+                .expect("Should insert genesis block without any problems");
+            db_transaction.commit().expect("Should commit");
+            db
+        }
+
+        fn add_message_to_relayer(db: &mut Database, message: Message) {
```

Comment:
It seems like this is adding a message to the database, but the name is `add_message_to_relayer`. Is this name right?

Reply:
With the https://github.com/FuelLabs/fuel-core/pull/1629, the name of the function will have more sense=) Since it will be `Database<Relayer>`